data/com/mojang/blaze3d/vertex/PoseStack.mapping

```diff
@@ -1,6 +1,8 @@
 CLASS com/mojang/blaze3d/vertex/PoseStack
 	METHOD mulPose (Lcom/mojang/math/Quaternion;)V
 		ARG 1 quaternion
+	METHOD mulPoseMatrix (Lcom/mojang/math/Matrix4f;)V
+		ARG 1 poseMatrix
```

Comment:
`poseMatrix` -> `matrix`. Mojang also uses this method to just rotate a matrix within the debug renderer so we cannot declare it is specifically for the pos.

Most cases would use the pose from some other matrix however. So we could put `pose`, but this is a precedent matter.